test/byContract/IexecPoco/verify.js

```diff
@@ -122,35 +122,35 @@ contract('Poco', async (accounts) => {
                 entry.sign,
             ),
         );
-        assert.isFalse(
-            await IexecInstance.verifyPresignatureOrSignature(
-                iexecAdmin.address,
-                entry.hash,
-                web3.utils.randomHex(64),
-            ),
-        );
-        assert.isFalse(
-            await IexecInstance.verifyPresignatureOrSignature(
+        const hex32 = 'ff'.repeat(32);
+        await expectRevert(
+            IexecInstance.verifyPresignatureOrSignature(
                 iexecAdmin.address,
                 entry.hash,
-                web3.utils.randomHex(64) + '1b',
+                '0x' + hex32 + hex32,
             ),
+            'ECDSAInvalidSignatureS("0x7f' + hex32.substring(2, hex32.length) + '")',
```

Comment:
Can you provide an example of this, please?

Reply:
Here is [Remove randomization in some verify.js signatures](https://github.com/iExecBlockchainComputing/PoCo/pull/55/commits/7fd7cbfd84343cc5665cd862f1b57cf321710e00) :heavy_check_mark:

Reply:
Nice thanks!